### PR TITLE
community/neovim: enable aarch64

### PR DESCRIPTION
**Pull request by z3ntu**
See NixOS/nixpkgs#61588

### CODE REVIEW

File: community/lua-bitop/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Contributor:
+# Maintainer:
```
Comment (maxice8) [inline community/lua-bitop/APKBUILD]: Why no maintainer ?

File: community/lua-bitop/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+pkgname="lua-bitop"
+pkgver="1.0.2"
+pkgrel=0
+pkgdesc="Bitwise operations on numbers"
+url="https://bitop.luajit.org/"
+arch="all"
+license="MIT"
+makedepends=""
```
Comment (maxice8) [inline community/lua-bitop/APKBUILD]: empty variables can be removed for brevity

**Comment by maxice8** [comment]
1. New package should start in testing then be moved to community/
2. New package has no maintainer

File: community/lua-bitop/APKBUILD (hunk 1 above)
Comment (z3ntu) [reply community/lua-bitop/APKBUILD]: I'd rather not put myself as maintainer for that package as I have little interest in lua and probably don't know how to fix any problems if they come up...

File: community/lua-bitop/APKBUILD (hunk 2 above)
Comment (z3ntu) [reply community/lua-bitop/APKBUILD]: Will do

**Comment by z3ntu** [comment]
> New package should start in testing then be moved to community/

As lua-bitop will be a dependency of neovim on aarch64, I don't think it can be in testing?

> New package has no maintainer

@jirutka You seem to be maintaining a lot of lua packages, would you maybe adopt this one?

File: community/neovim/APKBUILD [hunk 3]
```diff
@@ -19,10 +19,15 @@ pkgdesc="Vim-fork focused on extensibility and agility"
 url="https://neovim.io"
 # s390x: needs luajit
 # aarch64: fails to build
-arch="all !s390x !aarch64"
+arch="all !s390x"
 license="Apache-2.0 custom"
 makedepends="cmake gettext-dev gperf libtermkey-dev libuv-dev libvterm-dev
-	lua5.1-lpeg lua5.1-mpack luajit-dev msgpack-c-dev unibilium-dev"
+	lua5.1-lpeg lua5.1-mpack msgpack-c-dev unibilium-dev"
+if [ "$CARCH" = "aarch64" ]; then
```
Comment (tcely) [inline community/neovim/APKBUILD]: Using `case "${CARCH}" in` allows for easier expansion to other architectures.

**Comment by tcely** [comment]
We have LuaJIT for `aarch64` is using that an option?

**Pull request closed without merging**